jobs/config.json

```diff
@@ -8937,7 +8937,7 @@
       "--cluster=",
       "--deployment=kubernetes-anywhere",
       "--env-file=jobs/platform/gce.env",
-      "--extract=ci/latest-1.7",
+      "--extract=ci/latest-bazel-1.7",
```

Comment:
I was looking at the builds available in the ci gsbucket and it looks like latest-bazel-1.7 is not available (1.9 and 1.0 are available). Is this expected?


`$ gsutil ls gs://kubernetes-release-dev/ci/latest-bazel*
gs://kubernetes-release-dev/ci/latest-bazel-1.10.txt
gs://kubernetes-release-dev/ci/latest-bazel-1.9.txt
gs://kubernetes-release-dev/ci/latest-bazel.txt
`
I am suspect the 1.7 and 1.8 tests are going to fail when trying to pull the binaries for kubelet and kubeadm.